server/controllers/applications.coffee

```diff
@@ -75,7 +76,14 @@ module.exports.install = (req, res, next) ->
             log.error err.toString()
             sendError res, err, 400
         else
-            res.send 200, {"drone": {"port": result.port}}
+            # send path to home if it's a static app
+            if result.type is 'static'
+                res.send 200, {"drone": {
+                    "type": result.type
+                    "path": result.dir
+                }}
```

Comment:
Oh sorry i apologize i will test it like this. Just started coffeescript, so still not very expert on the syntax